libuvccamera/src/main/java/com/serenegiant/usb/USBMonitor.java

```diff
@@ -217,7 +217,7 @@ public synchronized void register() throws IllegalStateException {
                 final IntentFilter filter = new IntentFilter(ACTION_USB_PERMISSION);
                 // ACTION_USB_DEVICE_ATTACHED never comes on some devices so it should not be added here
                 filter.addAction(UsbManager.ACTION_USB_DEVICE_DETACHED);
-                context.registerReceiver(mUsbReceiver, filter);
+//                context.registerReceiver(mUsbReceiver, filter);
```

Comment:
Since now we have a git history, we don't need to "keep" the removed code, let's just remove the line(s) :)